defaults/main.yml

```diff
@@ -593,3 +593,22 @@ audit_results: |
       The pre remediation results are: {{ pre_audit_summary }}.
       The post remediation results are: {{ post_audit_summary }}.
       Full breakdown can be found in {{ goss_out_dir }}
+
+######### Headspace Customizations #############
+rhel7cis_hscustom: true
```

Comment:
Probably these should be in our main Ansible tree, instead of this public repo?

Reply:
gotta have defaults!  They'll be there too don't worry

Reply:
unless you think we shouldn't have defaults